demisto_sdk/commands/validate/tests/PB_validators_test.py

```diff
@@ -226,6 +229,48 @@ def test_IsAskConditionHasUnhandledReplyOptionsValidator():
     assert IsAskConditionHasUnhandledReplyOptionsValidator().is_valid([playbook])
 
 
+def test_PB125_playbook_only_default_next_valid():
+    """
+    Given:
+    - A default standard playbook.
+
+    When:
+    - calling PlaybookOnlyDefaultNextValidator.is_valid.
+
+    Then:
+    - The results should be empty as expected without validation error results.
+    """
+    playbook = create_playbook_object()
+    assert not PlaybookOnlyDefaultNextValidator().is_valid([playbook])
+
+
+def test_PB125_playbook_only_default_next_not_valid():
+    """
+    Given:
+    - A playbook with a condition task with only a default nexttask.
+
+    When:
+    - calling PlaybookOnlyDefaultNextValidator.is_valid.
+
+    Then:
+    - The results should contain a validation error object.
+    """
+    playbook = create_playbook_object()
+    playbook.tasks = {
+        "0": TaskConfig(
+            **{
+                "id": "test task",
+                "taskid": "27b9c747-b883-4878-8b60-7f352098a631",
+                "type": "condition",
+                "message": {"replyOptions": ["yes"]},
+                "nexttasks": {"#default#": ["1"]},
+                "task": {"id": "27b9c747-b883-4878-8b60-7f352098a63c"},
+            }
+        )
+    }
```

Comment:
Why not putting this under create_playbook_object?

Reply:
Because I want to test a regular case where it is valid, I'm adding the non-valid case.